healthbuddy_backend/rapidpro/tasks.py

```diff
@@ -4,7 +4,24 @@
 from django.conf import settings
 from django.utils import timezone
 
-from .models import Flow, DailyFlowRuns, Group, DailyGroupCount, DailyChannelCount, Channel
+from .models import (
+    Flow,
+    DailyFlowRuns,
+    Group,
+    DailyGroupCount,
+    DailyChannelCount,
+    Channel,
+    Label,
+    DailyLabelCount,
+)
+
+
+def get_all_results(next_call, headers, *args):
```

Comment:
The code is very good, congratulations. However i'm in doubt about, why was `*args` declared if it is not being used at any time? Do you think about using it in the future?

Reply:
Thanks!

To be honest, I copied this code snippet from another solution I made. In fact `*args` is not being used in this implementation, so I am going to remove it.